ci/job.yml

```diff
@@ -113,15 +113,79 @@
             echo ${sha1}
             echo ${GIT_URL}
             echo ${GIT_BRANCH}
+            CI_DEBUG=0
             if [ ${ghprbTargetBranch} == "openshift" ]
             then
               git rebase origin/${ghprbTargetBranch}
               cd ci
-              sh trigger_ci.sh ${GIT_URL} ${GIT_BRANCH} ${sha1}
+              sh trigger_ci.sh ${GIT_URL} ${GIT_BRANCH} ${sha1} ${CI_DEBUG}
             else
               echo "PR's target branch is not openshift"
             fi
 
+
+- job:
+    name: 'centos-container-pipeline-service-ci-pr-functional-debug'
+    description: |
+        Managed by Jenkins Job Builder, do not edit manually!
+    node: container
+    ci_project: container
+    properties:
+        - github:
+            url: https://github.com/CentOS/container-pipeline-service/
+    triggers:
+        - github-pull-request:
+            admin-list:
+                - kbsingh
+                - bamachrn
+                - dharmit
+                - navidshaikh
+                - mohammedzee1000
+                - cdrage
+            white-list:
+                - kbsingh
+                - bamachrn
+                - dharmit
+                - navidshaikh
+                - mohammedzee1000
+                - cdrage
+            trigger-phrase: '#dotests-debug'
+            only-trigger-phrase: false
```

Comment:
@bamachrn : We need to set this to `true`, so that, this debug job doesn't trigger on every PR but only when it finds the comment as defined.